code/modules/projectiles/guns/energy/railgun.dm

```diff
@@ -9,6 +9,7 @@
 	fire_sound = 'sound/weapons/rail.ogg'
 	item_charge_meter = TRUE
 	w_class = ITEM_SIZE_BULKY
+	extra_bulk = 2
```

Comment:
adding 2 bulk is the equalivant of making it have a small items more size when in modular storage.
This is fine